aws_auth.tf

```diff
@@ -38,22 +38,31 @@ locals {
     }
   ]
 
+  auth_fargate_roles = [
+    for index in range(0, var.create_eks && var.create_eks_fargate ? 1 : 0) : {
+      worker_role_arn = module.fargate.iam_role_arn
+      platform        = "fargate"
+    }
+  ]
+
```

Comment:
Add this as output in the fargate submodule like https://github.com/terraform-aws-modules/terraform-aws-eks/blob/master/modules/node_groups/outputs.tf#L6-L14